components/brave_shields/browser/ad_block_service.cc

```diff
@@ -95,15 +119,16 @@ void AdBlockService::OnComponentReady(const std::string& component_id,
 }
 
 void AdBlockService::OnResourcesFileDataReady(const std::string& resources) {
-  g_brave_browser_process->ad_block_service()->AddResources(resources);
+  AddResources(resources);
   g_brave_browser_process->ad_block_custom_filters_service()->AddResources(
```

Comment:
access custom_filters_service directly and remove the header for brave_browser_process